substrate/frame/revive/src/exec.rs

```diff
@@ -1135,7 +1149,7 @@ where
 			let contract = frame.contract_info.as_contract();
 			frame
 				.nested_storage
-				.enforce_subcall_limit(contract)
+				.enforce_limit(contract)
```

Comment:
The whole block above (github doesnt let me mark it):
```rust
if self.frames.is_empty() {
...
}
```

Is made redundant buy this change. You can just delete it.